bin/maintenance/build-wheel.py

```diff
@@ -310,8 +324,9 @@ def build_plugin(obj, assets, plugin_dir, add_version_suffix, ignore_unclean):
 @click.argument('plugins_dir', type=click.Path(exists=True, file_okay=False, resolve_path=True))
 @click.option('--add-version-suffix', is_flag=True, help='Add a local suffix (+yyyymmdd.hhmm.commit) to the version')
 @click.option('--ignore-unclean', is_flag=True, help='Ignore unclean working tree')
+@click.option('--assume-clean', is_flag=True, help='Assume clean working tree')
```

Comment:
The change seems reasonable. Just wondering about this naming. What about something like `--no-git-check` or `--no-git` which seems clearer?

Using `--add_version_suffix` together with that option should probably fail as well, because getting the suffix requires git as well.

Reply:
Hi @ThiefMaster ,
did that. Also, hopefully setting add_version_suffix=False with no_git makes sense.

Andrii